aspnetcore/fundamentals/servers/yarp/index.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: fundamentals/servers/yarp/index
+title: Welcome to the docs
+description: Welcome to the docs
+author: rick-anderson
+ms.author: riande
+ms.date: 02/06/2025
+ms.topic: article
+content_well_notification: AI-contribution
+ai-usage: ai-assisted
+---
+
+# Welcome to the docs
+
+This is the documentation for YARP 2.0.
+
+For documentation of YARP 1.1.1, see https://github.com/microsoft/reverse-proxy/tree/release/1.1/docs/docfx/articles.
+
```

Comment:
```suggestion
```

1.* is out of support now.

Reply:
@MihaZupan  Thanks for the help.
this is the 1st pass of the migration. This PR aims to minimize changes and just fix build errors and warnings. Next pass we'll fix up the doc